MathLib/LinAlg/Eigen/EigenMapTools.h

```diff
@@ -0,0 +1,190 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#ifndef MATHLIB_EIGENMAPTOOLS_H
+#define MATHLIB_EIGENMAPTOOLS_H
+
+#include <cassert>
+#include <vector>
+#include <Eigen/Core>
+
+namespace MathLib
+{
+/*! Creates an Eigen mapped matrix from the given data vector.
```

Comment:
Needs improvement, because the data vector is also changed. The way it is now, suggests that the data vector is not changed.